packages/clerk-js/src/core/constants.ts

```diff
@@ -37,3 +37,6 @@ export const SIGN_UP_MODES: Record<string, SignUpModes> = {
   PUBLIC: 'public',
   RESTRICTED: 'restricted',
 };
+
+// This is the currently support version of the Frontend API
+export const SUPPORTED_FAPI_VERSION = '2024-10-01';
```

Comment:
I'm curious about FAPI versioning and keeping types in sync. The types package version and which FAPI version they represent.